File: android/java/app/src/main/java/com/daily/partyline/WebAppClientCallback.java
package com.daily.partyline;

public interface WebAppClientCallback {

    void setRoomName(String roomName);
    void onJoinedMeeting();
    void onTick(String timer);
    void onDataChanged();
    void onForceEject();
    void onError();
    void onRoleChanged();
    void onEndCall();
    void onAudioStateChanged(Boolean isMuted);
}

File: android/kotlin/app/src/main/java/com/daily/partyline/WebAppClientCallback.kt
package com.daily.partyline

interface WebAppClientCallback {
    fun setRoomName(roomName: String?)
    fun onJoinedMeeting()
    fun onTick(timer: String?)
    fun onDataChanged()
    fun onForceEject()
    fun onError()
    fun onRoleChanged()
    fun onEndCall()
    fun onAudioStateChanged(isMuted: Boolean)
}